extensions/amp-image-slider/0.1/test/integration/test-amp-image-slider.js

```diff
@@ -446,5 +470,95 @@ config.run('amp-image-slider', function() {
             return pollByBarLeftPos(rect.right);
           });
     });
+
+    it('should seekTo correct position', () => {
+      let button;
+      let pos10Percent;
+      return prep()
+          .then(() => {
+            button = doc.querySelector('#button');
+            pos10Percent = rect.left + 0.1 * rect.width;
+            button.click();
+            return pollByBarLeftPos(pos10Percent);
+          });
+    });
+
+    it('should show hint again on scroll back and into viewport', () => {
+      let slider, container_, hint;
```

Comment:
One variable per declaration (https://google.github.io/styleguide/jsguide.html#features-one-variable-per-declaration)

Reply:
Got it.